added domain-setup.ftd

nandhinidevie opened the pull request:
@Heulitig please review and merge

amitu commented on workshop/learn.ftd:
```diff
@@ -57,6 +57,7 @@ offered to the students to acknowledge their competance in `fastn-stack`.
 - **Static hosting on GitHub pages:** Learn to [publish static site on GitHub Pages](/github-pages/)
 - **Static hosting on Vercel:** Learn to [publish static site On Vercel](/vercel/)
 - **Dynamic hosting:** Learn [deploying on Heroku](/heroku/)
+- **Setup your domain:** Learn how to [setup a domain](/workshop/domain-setup/) for your website.
```
Also add this FASTN.ftd under `Docs -> Deploy`.

amitu commented on workshop/domain-setup.ftd:
```diff
@@ -0,0 +1,280 @@
+
+- In Name field type @ 
+
+- In Value enter 185.199.108.153 ip address([refer Step No.2 from this document](https://docs.google.com/document/d/1aD_nol3-4QSQW4d5Tn9BgpwbDj58eQAMtbiBtZCB5I8/edit#bookmark=id.ohc2t8qa8z6x))
```
Link to this instead: https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain

amitu commented:
@nandhinidevie Some of the images you have added are already in our repo in other places where we tell people how to create a new site from our template. I think that part is duplicated a lot right now, maybe we can convert that to component and use it everywhere?

nandhinidevie commented:
> @nandhinidevie Some of the images you have added are already in our repo in other places where we tell people how to create a new site from our template. I think that part is duplicated a lot right now, maybe we can convert that to component and use it everywhere?

@amitu, Converted the steps into a component (component name: create-site-using-fastn-template) and added it to the content-library/index.ftd
